technical_documentation/frontend_service.md

```diff
@@ -9,6 +9,8 @@ Furthermore, it is common practice to not strictly separate HTML and JS in diffe
 ## Functionality explained by Actions
 ### Create or edit a Routine
 After the login a user is able to either edit existing routines or create a new one from scratch. Through the model binding (binding a JSON to specific input fields of the HTML), there is no specical effort in creating a valid JSON for a routine. Through the two-way binding concept of VueJS, all the necessary information is handed-in by the user automatically by filling the required input fields. The information about the available components and their metadata is described in the [master service documentation, section AVAILABLE_COMPONENTS](https://github.com/D067659/tpmc_documentation/blob/master/technical_documentation/master_service.md#available_components-endpoint).
+A JSON representaion of a succesfully saved routine can be found in the [exemplary_function_fixtures - routine section](https://github.com/D067659/tpmc_documentation/blob/master/technical_documentation/exemplary_function_fixtures/routine-example.txt).
```

Comment:
Fehler in der URL: der file heißt routine_example.txt. Eventuell würde ich für den File auch einen neuen Ordner "exemplary_routine_fixtures" erstellen.

Reply:
Würde einen Verweis auf diesen File zusätzlich auch noch in die Master Service Doku reinnehmen.

Reply:
Typo in "representation"

Reply:
done!